ambari-common/src/main/python/resource_management/libraries/functions/curl_krb_request.py

```diff
@@ -124,7 +124,7 @@ def curl_krb_request(tmp_dir, keytab, principal, url, cache_file_prefix,
     # kinit if it's time; this helps to avoid problems approaching ticket boundary when
     # executing a klist and then a curl
     last_kinit_time = _KINIT_CACHE_TIMES.get(ccache_file_name, 0)
-    current_time = long(time.time())
+    current_time = long(time.time() * 1000)
```

Comment:
lets define a function in the code like:

```
def get _current_time_ms():
  return round(time.time() * 1000)
```
and then use it instead any occurrence of `time.time()` in the module ?